libraries/elementresources/src/main/res/values/theme_dark.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2022 New Vector Ltd
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
```

Comment:
Why do we need this file?

Reply:
The surface and other material colours won't be properly applied in dark mode otherwise, making the app impossible to use (@kittykat reported this):

<summary>Screenshot of login screen in dark mode:</summary>
<details> 
![Screenshot_1677495436](https://user-images.githubusercontent.com/480955/221546032-601a6972-ac89-4271-8b8f-74c38a476a30.png)
</details>

Reply:
It shouldn't have impact as it's not used on a full-compose themed app???

Reply:
Maybe it's caused by the `postSplashScreenTheme` theme attr?

Reply:
Removing what exactly? we shouldn't remove `postSplashScreenTheme`. 
I think the issue is more related to the `Box` background which was not applied in the `MainActivity`

Reply:
> Removing what exactly? we shouldn't remove `postSplashScreenTheme`. I think the issue is more related to the `Box` background which was not applied in the `MainActivity`

I think you're right. I tried changing the `<style name="Theme.ElementX" parent="Theme.ElementAndroid" />` in `values-night` to point at the Light theme instead once we have the root `Box` with a proper background and everything worked fine except for the composer, which seems to be broken with the `.Dark` theme and the default one, so it's probably not related?

<img width="660" alt="image" src="https://user-images.githubusercontent.com/480955/221603869-6da50f7c-42ee-4000-9aaa-1c6ad874ffc1.png">

Reply:
Yeah composer is not correctly handled, we should fix that on an other PR